src/command-registry.coffee

```diff
@@ -18,6 +18,11 @@ SequenceCount = 0
 # command event listeners globally on `atom.commands` and constrain them to
 # specific kinds of elements with CSS selectors.
 #
+# Command names must follow the `namespace:event` pattern, where both parts can
+# be further split into words by hyphens. Typically `namespace` will be the
```

Comment:
Well, like configuration settings ... I don't think we want to make it an absolute requirement. I prefer to leave some things as "Ok, if you need to break the rules ... that's fine, but it's ice skating uphill".